scene/register_scene_types.cpp

```diff
@@ -583,6 +585,7 @@ void register_scene_types() {
 	GDREGISTER_CLASS(CPUParticles3D);
 	GDREGISTER_CLASS(Marker3D);
 	GDREGISTER_CLASS(RootMotionView);
+	GDREGISTER_ABSTRACT_CLASS(SkeletonModifier3D);
```

Comment:
```suggestion
	GDREGISTER_CLASS(SkeletonModifier3D);
```

Reply:
https://github.com/godotengine/godot/pull/87888#discussion_r1540729096

Same as above comment. Perhaps another class or something for extensions should be discussed later to be published.

Reply:
This isn't about extensions. The problem is with GDScript, since GDScript scripts do not register with ClassDB, they are considered instances of the base class directly.

Reply:
Yes, so I thought we were talking about not allowing this as a GDScript. Is that correct @reduz ?

Perhaps a template or something should be provided to allow easy creation of GDExtention.